catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/EnumerationValidator.java

```diff
@@ -37,6 +37,8 @@
  * Validates an attribute's value(s) against a set of acceptable values.
  */
 public class EnumerationValidator implements AttributeValidator {
+    private final boolean ignoreCase;
```

Comment:
I am not sure if I agree.  An `EnumerationValidator` that is given a list of values that are all lower case and ignore case false has the same values and behaviors as one given values of mixed case and ignore case true.  

This private field can and should be removed since ignore case is used only in the constructor.